Automattic/jetpack
# Shortcodes: ensure that the Spotify shortcode handles lazy loading

Fixes #16454

#### Changes proposed in this Pull Request:

* Added lazy loading attribute in the `<iframe>` tag.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use? NO



#### Testing instructions:






*  Go to your WordPress
*  Start creating a new post
*  Write a long text in the content of the post ( You can take the help of lorem ipsum)
* At the end of long text add shorcode something like this `[spotify https://open.spotify.com/album/4BC7xFBCxUMBEgGpxRBaCy width="400" height="100"]`
*  Save and publish that post.
*  Open published  post on google-chrome 
*  Open developer tools and reload that page
*  Open `Network Tab` and observe network calls
*  Initially there will be no calls to `*.spotify.com`
*  Keep scrolling that post and you will see that `Spotify` network call is later when it is coming in the current viewport.

#### Benefits of implementing lazy loading: 
* Based on **web.dev** we can **reduce initial load time**
*  Learn more about lazy loading: https://web.dev/iframe-lazy-loading/


#### Screen recoding: https://www.youtube.com/watch?v=NdMFOfzEaOY
